arviz/plots/backends/bokeh/distplot.py

```diff
@@ -105,6 +114,10 @@ def _histplot_bokeh_op(values, values2, rotated, ax, hist_kwargs):
         hist_kwargs["fill_color"] = color
         hist_kwargs["line_color"] = color
 
+    line_alpha = hist_kwargs.pop("line_alpha", False)
+    if not line_alpha:
+        hist_kwargs["line_alpha"] = 0
```

Comment:
I think this behaves like `hist_kwargs.setdefault("line_alpha", 0)`? Have not checked though